backend/src/api/bs/bs.api.ts

```diff
@@ -137,6 +138,29 @@ const initBsApi: FastifyPluginAsync<Options> = async (fastify, opts) => {
       return rep.send(object).status(HttpCode.OK);
     },
   });
+
+  fastify.route({
+    method: HttpMethod.GET,
+    url: `${BSApiPath.SPACES}${SpacesApiPath.$ID_OBJECTS}`,
+    async handler(
+      req: FastifyRequest<{
+        Querystring: BSObjectGetRequestParamsDto;
+        Params: { id: string };
+      }>,
+      rep,
+    ) {
+      const [, token] = req.headers?.authorization?.split(' ') ?? [];
+
+      const spaces = await bsObjectService.getObjects({
+        spaceId: req.params.id,
+        from: req.query.from,
+        count: req.query.count,
+        token,
+      });
+
+      return rep.send(spaces).status(HttpCode.OK);
```

Comment:
maybe better name objects or spaceObjects?
```suggestion
      const objects = await bsObjectService.getObjects({
        spaceId: req.params.id,
        from: req.query.from,
        count: req.query.count,
        token,
      });

      return rep.send(objects).status(HttpCode.OK);
```